design/templates/solution_design.md

```diff
@@ -67,5 +71,41 @@
 ## Open questions
 [//]: # (Add any question that is still open. It makes it easier for the reader to have the open questions accumulated here istead of them being acattered along the doc)
 
-## Implementation plan
+## Delivery plan
 [//]: # (Break the solution into tasks)
+
+### Total
+[//]: # (Write the total estimation)
+
+### Parallelism
+[//]: # (Suggest how many team members can work on the solution in parallel)
+
+### Ramp up
+[//]: # (Estimate the tasks)
```

Comment:
I'm not sure what "ramp up" means - is this about the prep / spike / planning work that would go into the effort?

Reply:
Actually we are using this phrase a lot, it means to learn: new technology, new language, new area in the product

Reply:
can you add this data to the description so it is clear for future users of this template?

Reply:
Done :)